backend/controllers/caseNote.ts

```diff
@@ -36,7 +36,7 @@ export const caseNoteFactory = (prisonApi, caseNotesApi) => {
       if (req.xhr) {
         const { typeCode } = req.query
         const filteredSubTypes = subTypes.filter((st) => st.type === typeCode)
-        return res.send(nunjucks.render('caseNotes/partials/subTypesSelect.njk', { subTypes: filteredSubTypes }))
+        return res.send(nunjucks.render('caseNotes/partials/subTypesOptions.njk', { subTypes: filteredSubTypes }))
```

Comment:
The select template included the full `<select>` tag which then meant I couldn't attach an `onchange` event to the sub types.  I've therefore created a new options template that just replaces all the `<option>`s instead